app/retail/templates/about.html

```diff
@@ -14,128 +14,402 @@
 <!DOCTYPE html>
 <html lang="en">
-  {% include 'shared/head.html' %}
+
   <head>
-    <meta name="title" content="About Gitcoin: Find Open Bug Bounties & Freelance Coders">
-    <meta name="description" content="Gitcoin is a community for freelance developers to collaborate and monetize their skills
-    while working on Open Source projects through bounties paid in crypto (ETH or any ERC-20 token).">
-    {% compress css file about %}
-      <link rel="stylesheet" type="text/x-scss" href={% static "v2/scss/landing_page.scss" %} />
-      <link rel="stylesheet" type="text/x-scss" href={% static "v2/scss/about.scss" %} />
-    {% endcompress %}
+    {% include 'shared/head.html' %}
+    {% include 'shared/cards_pic.html' %}
+    <style>
+      @property --grad {
+        syntax: '<angle>';
+        initial-value: 0deg;
+        inherits: false;
+      }
+      .grad {
+        --grad: 280deg;
+        width: 100%;
+        padding: 4em 0;
+        will-change: background;
+        background: radial-gradient(78.19% 34.75% at 90.21% 104.67%, #FFFFFF 54.6%, rgba(255, 255, 255, 0) 100%), radial-gradient(65.42% 69.3% at 47.67% 83.08%, #FFFFFF 0%, #FFFFFF 50.74%, rgba(255, 255, 255, 0) 100%), radial-gradient(31.85% 86.5% at 3.3% 92.42%, #FFFFFF 42.79%, rgba(255, 255, 255, 0) 100%), conic-gradient(from var(--grad) at 50% 50%, #FFBEE7 -39.94deg, #FFDDF1 39.37deg, #F5E1F4 129.16deg, #EFD3FE 153.75deg, #DCC0FF 176.25deg, #F3DBFF 233.58deg, #FFBEE7 320.06deg, #FFDDF1 399.38deg);
+        animation: bg-space-animation 15s ease-in-out infinite;
+      }
+
+      .bubble-wrap {
+        overflow: hidden;
+        height: 600px;
+      }
+
+      .bubbles {
+        position: relative;
+        transition: opacity linear .3s;
+      }
+
+      .bubble {
+        position: absolute;
+        width: 152px;
+        height: 152px;
+        border-radius: 50%;
+        will-change: transform;
+        object-fit: cover;
+        box-shadow:
+          0 15px 35px rgba(0, 0, 0, 0.1),
+          0 3px 10px rgba(0, 0, 0, 0.1);
+      }
+
+      @keyframes bg-space-animation {
+        from {
+          --grad: 0deg;
+        }
+        to {
+          --grad: 360deg;
+        }
+      }
+
+      @keyframes scroll {
+        /* 0% { transform: translateX(0); } */
+        100% { transform: translateX(calc(var(--cardSize ) * var(--cardCount) * var(--direction)))}
+      }
+
+      .auto-user-slide {
+        animation: scroll var(--time) linear infinite;
+        display: flex;
+        width: calc(var(--cardSize) * var(--cardCount));
+        will-change: transform;
+      }
+
+      .kernel-slide {
+        --time:90s;
+        --direction: -1;
+        --cardSize: 318px;
+        --cardCount: {{kernel|length}};
+      }
+
+      .users-slide {
+        --time:60s;
+        --direction: -1;
+        --cardSize: 318px;
+        --cardCount: {{data.top_earners|length}};
+        animation-direction: reverse;
+      }
+
+
+
+    </style>
```

Comment:
^ worth moving it to it's own file ?

Reply:
lets do it on the future Im worry about some special "css" we are using here to introduce some problems with our bundle 
@gdixon we should do some testing about @property Im pretty sure will get into "sass" without being sass